File: src/mongodb/collections/Starboards.cs
using System.Runtime.Serialization;
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Driver;

public class Starboards
{
    [BsonId]
    [DataMember]
    public MongoDB.Bson.ObjectId _id { get; set; }
    /// <summary>
    /// The ID of the current message. Used for fetching the embed message on Discord.
    /// </summary>
    public ulong MessageId;
    /// <summary>
    /// The ID of the message that was starred.
    /// </summary>
    public ulong MessageStarredId;
}

File: apps/discord-bot/app/src/main/java/discordbot/mongodb/collections/Starboards.java
package discordbot.mongodb.collections;

import org.bson.codecs.pojo.annotations.BsonId;
import org.bson.types.ObjectId;

import kotlin.ULong;

public class Starboards {
    @BsonId
    public ObjectId _id;
    /**
     * The ID of the current message. Used for fetching the embed message on Discord.
     */
    public ULong MessageId;
    /**
     * The ID of the message that was starred.
     */
    public ULong MessageStarredId;
}
